.jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat

```diff
@@ -63,26 +64,23 @@ set NUMBAPRO_NVVM=%CUDA_PATH%\nvvm\bin\nvvm64_32_0.dll
 
 set PYTHONPATH=%TMP_DIR_WIN%\build;%PYTHONPATH%
 
-if NOT "%BUILD_ENVIRONMENT%"=="" (
-    pushd %TMP_DIR_WIN%\build
-    copy /Y %PYTORCH_FINAL_PACKAGE_DIR_WIN%\%IMAGE_COMMIT_TAG%.7z %TMP_DIR_WIN%\
-    :: 7z: -aos skips if exists because this .bat can be called multiple times
-    7z x %TMP_DIR_WIN%\%IMAGE_COMMIT_TAG%.7z -aos
-    popd
-) else (
-    xcopy /s %CONDA_PARENT_DIR%\Miniconda3\Lib\site-packages\torch %TMP_DIR_WIN%\build\torch\
+:: Install pre-built bdist_wheel
+FOR %%A IN (%PYTORCH_FINAL_PACKAGE_DIR%\*.whl) DO (
+    call python -m pip install %%A
+    if %errorlevel% neq 0 ( exit /b %errorlevel% )
 )
+@echo off
+for /f %%W in ('python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"') DO set site_packages=%%w
```

Comment:
This is setting the `site_packages` variable, don't ask me how it does this in batch, I don't really know